plugins/leemons-plugin-users/frontend/src/components/UserDetail/UserDetail.js

```diff
@@ -26,21 +27,30 @@ export const USER_DETAIL_VIEWS = {
 function UserDetail({
   userId,
   center,
+  hideTags,
   viewMode = USER_DETAIL_VIEWS.ADMIN,
   sysProfileFilter,
   onLoadUser = noop,
+  onLoadUserAgents = noop,
+  onChangeAvatar = noop,
+  canEdit,
 }) {
   const enableUserDetails = !!userId;
   const { data: userDetails, isLoading } = useUserDetails({
     userId,
     enabled: enableUserDetails,
   });
   const [t] = useTranslateLoader(prefixPN('user_detail'));
+  const [tForm, , , tFormLoading] = useTranslateLoader(prefixPN('userForm'));
```

Comment:
```suggestion
  const [tForm] = useTranslateLoader(prefixPN('userForm'));
```

tFormLoading is not being used